contracts/token/RarePizzasBox.sol

```diff
@@ -171,10 +173,15 @@ contract RarePizzasBox is
             require(toPaisanos[i] != address(0), 'dont be silly');
             _presaleAllowed[toPaisanos[i]] = count;
         }
+
+        emit PresaleAllowedUpdated('setPresaleAllowed was called');
```

Comment:
lets capture the `count` here as a the param. no need for the string since the event signature is also posted

Reply:
Thought of that, my reasoning was. Just a count alone doesn't say much if you don't know what it was before or how many addresses are involved. In that case I would go with the empty event.